project_tracker_template/update_status.gs

```diff
@@ -0,0 +1,46 @@
+/**
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Updates status cells' values to 'Not Started','Done' or 'In Progress' in the
+ * Task Sheet
+ * @param {Object[][]} taskTable A two dimensional array containing values of
+ *     the table in the Task Sheet
+ * @param {Number} tableLength Length of taskTable array
+ */
+function updateStatus(taskTable, tableLength) {
+  for (var i = 0; i < tableLength; i++) {
+    var estimatedDays = taskTable[i][8];   // Column I
```

Comment:
have names for the columns as constants (not I, J etc byt something like ESTIMATED_DAYS_COLUMN etc)